### DISCUSSION TITLE
Explore using datatree for model eval

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [hytest-org/hytest#258] [Coding Tips and Tricks]
xarray datatree seems like it would be perfect for model eval or whenever you have more than one dataset you are trying to do something with.  

Here's the link to the 15 min Pangeo Showcase Talk on DataTree from a few weeks ago: 
https://www.youtube.com/watch?v=ST6CrJ58bkI
https://zenodo.org/record/7679730#.Y_zHEibMJPY

**Comment by rsignell-usgs** [comment]
ping @sfoks , @thodson-usgs , @alaws-USGS

**Comment by alaws-USGS** [comment]
@rsignell-usgs I watched the video and I can see that being good for model eval. For tutorial work, I would say it would be a good advanced tutorial concept to teach. Once a script is going to be run for a full model evaluation, this could definitely help shorten the code.
